### ISSUE TITLE
Unexpected $PATH causing git hooks/pgp signing to fail with v10.0.52.2

### ISSUE THREAD

**Issue by edwardloveall** [ForkIssues/Tracker#71]
Hello 🙂

I have a git hook (commit-msg) that looks like this:

```shell
fold -s -w 72 "$1" | sponge "$1"
```

It makes every line 72 characters or less (cut off at a space) and then pipes that output back into it self. When I commit anything in v10.0.52.2, I get this message:

```
.git/hooks/commit-msg: line 1: sponge: command not found
```

Sponge is part of the [moreutils](https://joeyh.name/code/moreutils/) package, installed via [homebrew](https://github.com/Homebrew/homebrew-core/blob/master/Formula/moreutils.rb).

If I disable (comment out) the commit-msg hook and try to commit again, I get this GPG error:

```
error: cannot run gpg: No such file or directory
error: gpg failed to sign the data
fatal: failed to write commit object
```

GPG is installed via https://gpgtools.org.

Both `gpg` and `sponge` are found because I've added their directories to my `$PATH`. If I inspect the path when committing in Fork, I see that it has it's own, different path which doesn't include the directories that have `sponge` or `gpg`.

So my _guess_ is it's a `$PATH` issue, possibly having to do with the bundled git in the latest version. But I'm sure you will know better than me 🙂 

Thanks for the app!

**Comment by DanPristupov** [comment]
@edwardloveall thank you for the well explained bug report. I'm investigating the problem.

**Comment by DanPristupov** [comment]
Hi @edwardloveall. I just released Fork 1.0.53 with the fix. Could you please confirm that it solves the problem when you have a chance to try it?

**Comment by edwardloveall** [comment]
Seems fixed to me! Thanks a bunch!
